### DISCUSSION TITLE
how to do smartcase search?

### DISCUSSION THREAD

**Discussion by laoshaw** [dandavison/delta#1786] [Q&A]
is this supported? say, I have 'git log -p' and I type '/keyword' and it will search keyword with smartcase?

**Comment by dandavison** [comment]
Hi @laoshaw, when you type `/` you are interacting with `less`, not `delta` (i.e. whatever version of `less` you have installed that `delta` is picking up). I'm not exactly sure what smartcase is but you can do whatever your version of less can do.
